model/keyvalue.go

```diff
@@ -50,54 +50,6 @@ const (
 	binaryTypeStr  = "binary"
 )
 
-func (p ValueType) String() string {
```

Comment:
this is just moving down in the file, to match the location of similar new methods for TraceID